packages/core-components/src/layout/Sidebar/Items.test.tsx

```diff
@@ -71,7 +71,7 @@ describe('Items', () => {
     it('should render a button with custom style', async () => {
       expect(
         await screen.findByRole('button', { name: /create/i }),
-      ).toHaveStyle(`background-color: ${hexToRgb('2b2a2a')}`);
+      ).toHaveStyle(`background-color: transparent`);
```

Comment:
This test was wrong from the start, `hexToRgb` expects a hex number with the symbol, so it should have been `hexToRgb('#2b2a2a')`, now with Button it should be transparent and let the container define the color.

Reply:
Visually it looks the same to me.